src/server/session.ts

```diff
@@ -2709,6 +2711,63 @@ export class Session<TMessage = string> implements EventSender {
         }
     }
 
+    private getMoveToRefactoringFileSuggestions(args: protocol.GetMoveToRefactoringFileSuggestionsRequestArgs): { newFilename: string | undefined, files: string[] | undefined }{
+        const { file } = this.getFileAndProject(args);

```

Comment:
I thought you wanted this only from the project that file default belongs to ? 
If not then we swould need loadAncestorProjectTree to get more projects to provide info from ?

Reply:
Yes, it's only for the project the file defaults to. I'll remove the other checks.